### PR TITLE
Grid-Layout-ONDER

### PR DESCRIPTION
**Pull request by ondermartin**

### CODE REVIEW

**Review by annasvst — COMMENTED** [review]
Hi @ondermartin, I think you created a new branch for the responsive-grid-layout exercise when being on the branch for pricing-page-exercise. 
When creating a new branch it's important to check that you are on master/main branch or that you check out from the master/main branch.

File: responsive-grid-layout/onder/index.html [hunk 1]
```diff
@@ -1,42 +1,18 @@
...
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" href="styles.css">
+<title>Grid Layout</title>
+</head>
+<body>
+<div class="container">
+<header>Header</header>
+<sidebar>sidebar-left</sidebar>
```
Comment (annasvst) [inline responsive-grid-layout/onder/index.html:12]: I haven't heard of the `sidebar` element. A `div` would be more appropriate here :)

File: responsive-grid-layout/onder/index.html [hunk 2]
```diff
@@ -1,42 +1,18 @@
...
+<main>Content</main>
+<aside>Sidebar-right</aside>
```
Comment (annasvst) [inline responsive-grid-layout/onder/index.html:14]: I think the aside is not perfect fit for a side bar, it's usually used for smaller things. Check this page: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside
My recommendation is to use a `div` here instead.

File: responsive-grid-layout/onder/styles.css [hunk 3]
```diff
@@ -0,0 +1,55 @@
+*{
+  margin: 0;
```
Comment (annasvst) [inline responsive-grid-layout/onder/styles.css:2]: I'd recommend to never set padding and margin to 0 for all elements (selector *)

File: responsive-grid-layout/onder/styles.css [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+   grid-template-columns: 1fr 4fr 1fr;
+   grid-template-rows: 80px 1fr 80px;
+   grid-template-areas:  
+   "header header header"
+   "sidebar-left content sidebar-right"
+   "sidebar-left footer footer"
+}
+
+header {
+   display: grid;
```
Comment (annasvst) [inline responsive-grid-layout/onder/styles.css:25]: For the sake of this exercise, I think you only need to set `display: grid;` for the `div` with class 'container'. All other elements don't really have contain that you'd need to align.

File: responsive-grid-layout/onder/styles.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+   background-color: #33b4;  
+   grid-template-columns: 1 / span 5;
```
Comment (annasvst) [inline responsive-grid-layout/onder/styles.css:27]: `grid-template-columns: 1 / span 5;` is one of the ways to make the layout look as we need it to. However, using `grid-area: header;` is a cleaner way to do that. You don't need to use both of them at the same time.

File: responsive-grid-layout/onder/styles.css [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+   grid-area: header;
+}
+
+sidebar{
+   display: grid;
```
Comment (annasvst) [inline responsive-grid-layout/onder/styles.css:32]: Same here about display and grid-template-areas.

File: responsive-grid-layout/onder/styles.css [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+  grid-template-areas: 1 / span 2;
+   grid-area: sidebar-left;
+   background-color: #56ffa5;
+   }
+
+ aside {
+   display: grid;
```
Comment (annasvst) [inline responsive-grid-layout/onder/styles.css:39]: Same here about `display`.

File: responsive-grid-layout/onder/styles.css [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+     background-color: #56ffa5;
+     grid-area: sidebar-right;
+}
+
+ main {
+   display: grid;
```
Comment (annasvst) [inline responsive-grid-layout/onder/styles.css:45]: Same here about `display`.

File: responsive-grid-layout/onder/styles.css [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+     grid-area: content;
+     background-color: red;
+  
+     }
+footer{
+   display: grid;
+   background-color: #ddd;
+   grid-column: 2 / 4 ;
```
Comment (annasvst) [inline responsive-grid-layout/onder/styles.css:53]: You don't need to specify `display` and `grid-column` here.

**Review by annasvst — COMMENTED** [review]
Well done, left some minor comments :)

File: responsive-grid-layout/onder/index.html (hunk 2 above)
Comment (ondermartin) [reply responsive-grid-layout/onder/index.html:14]: Okay thank you alot for this hint

File: responsive-grid-layout/onder/styles.css (hunk 3 above)
Comment (ondermartin) [reply responsive-grid-layout/onder/styles.css:2]: thank you again

File: responsive-grid-layout/onder/styles.css (hunk 4 above)
Comment (ondermartin) [reply responsive-grid-layout/onder/styles.css:25]: Tugba and I actually were doing the project together but both of us was wrong. good to know. We will correct all of that for the next assignment.

File: responsive-grid-layout/onder/styles.css (hunk 6 above)
Comment (ondermartin) [reply responsive-grid-layout/onder/styles.css:32]: copy

Comment (ondermartin) [reply responsive-grid-layout/onder/styles.css:32]: I copy that

File: responsive-grid-layout/onder/styles.css (hunk 9 above)
Comment (ondermartin) [reply responsive-grid-layout/onder/styles.css:53]: I copy that

**Review by annasvst — APPROVED** [review]
Approved!

**Pull request merged**